test/components/baseAxisTests.ts

```diff
@@ -196,16 +196,4 @@ describe("BaseAxis", () => {
 
     svg.remove();
   });
-
-  it("default alignment based on orientation", () => {
```

Comment:
I agree about touching the underscored variables. Can we test this by calling `renderTo()` on axes in a square `<svg>` and testing where their bounding boxes wind up?